dotnet/Templates/Microsoft.MacCatalyst.Templates/maccatalyst-view/.template.config/localize/templatestrings.es.json

```diff
@@ -1,6 +1,6 @@
 {
   "author": "Microsoft",
-  "name": "Vista de Mac\u00A0Catalyst",
+  "name": "Vista de Mac Catalyst",
```

Comment:
@tj-devel709 is this expected now?

Reply:
Commit from last week removing the non-breaking space: https://github.com/xamarin/xamarin-macios/pull/18272/files

Git Blame does not show that this commit happened..
https://github.com/xamarin/xamarin-macios/blame/main/dotnet/Templates/Microsoft.MacCatalyst.Templates/maccatalyst-view/.template.config/localize/templatestrings.es.json

Looking at this PR on VSCode, it looks like the non-breaking space is removed as well so I would say let's try it and see if the change goes through this time